xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -616,6 +620,22 @@ rebuild_master:
             repository: openSUSE_Tumbleweed
             architecture: x86_64</screen>
         </sect4>
+
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.obs_workflows.steps.trigger_services">
+          <title>Trigger Services of a Package</title>
+
+          <para>Given a project called <emphasis>home:Admin</emphasis> and a
+          package <emphasis>ctris</emphasis>, the step will trigger services of the
+          package <emphasis>home:Admin/ctris</emphasis>.</para>
+
```

Comment:
What about adding the following sentence?

"Make sure the `ctris` package contains a correct `_service` file. Refer to [obs-service-tar_scm](https://github.com/openSUSE/obs-service-tar_scm#user-documentation) for more details. "

Reply:
I've pushed a change to mention the `_service` file. Let me know what you think.